lynx_description/launch/load_urdf.launch.py

```diff
@@ -177,6 +184,7 @@ def generate_launch_description():
         package="joint_state_publisher",
         executable="joint_state_publisher",
         namespace=namespace,
+        arguments=["--ros-args", "--log-level", log_level, "--log-level", "rcl:=INFO"],
```

Comment:
Maybe unnecessary for this node. Up to you

Reply:
Removed